src/assets/sass/protocol/components/_navigation.scss

```diff
@@ -169,6 +169,15 @@
     }
 }
 
+.mzp-c-navigation-menu-button:not(.has-label) {
```

Comment:
Not a blocker, but could we improve legacy browser support a bit if we made the default not having a label? (removing the reliance on `:not()`?

Reply:
I ran into some problems over-riding the image replacement when I did it the other way. I can try again but do we even serve IE8 these files?

Reply:
I don't think IE8 would ever seen the mobile menu button, I was thinking more of the long tail of older mobile devices out there. I took a look on [caniuse](https://caniuse.com/#feat=css-sel3) and support seems pretty broad. I think we're likely fine to use it 👍